src/utils/getTodoFilter.ts

```diff
@@ -0,0 +1,17 @@
+import { Filter } from '../types/Filter';
+import { Todo } from '../types/Todo';
+
+export const getTodoFilter = (todos: Todo[], filter: Filter) => {
```

Comment:
`getTodoFilter` sounds like this functions returns filter. So instead rename it:
```suggestion
export const getFilteredTodos = (todos: Todo[], filter: Filter) => {
```